src/DataLoader.js

```diff
@@ -24,17 +24,46 @@ var groupEvidencesByCode = function(features) {
     return features;
 };
 
+var setVariantData = function (authority, d) {
```

Comment:
I don't understand what is happening here, could you explain why all of d's attributes are copied to datum and then d is added to `datum.externalData[authority]`?